packages/augur-tools/src/libs/ganache.ts

```diff
@@ -90,7 +90,7 @@ async function makeGanacheOpts(accounts: Account[], db: MemDown) {
     // TODO: For some reason, our contracts here are too large even though production ones aren't. Is it from debugging or lack of flattening?
     allowUnlimitedContractSize: true,
     db,
-    gasLimit: 75000000000,
+    gasLimit: 100000000000,
```

Comment:
```suggestion
    gasLimit: 10000000,
```

make ganache have the same limit as everything else